doc/source/user_guide/groupby.rst

```diff
@@ -1303,16 +1307,16 @@ column ``B`` because it is not numeric. We refer to these non-numeric columns as
    df.groupby("A").std(numeric_only=True)
 
 Note that ``df.groupby('A').colname.std().`` is more efficient than
-``df.groupby('A').std().colname``, so if the result of an aggregation function
-is only interesting over one column (here ``colname``), it may be filtered
+``df.groupby('A').std().colname``. So if the result of an aggregation function
+is only needed over one column (here ``colname``), it may be filtered
 *before* applying the aggregation function.
 
 .. note::
-   Any object column, also if it contains numerical values such as ``Decimal``
-   objects, is considered as a "nuisance" column. They are excluded from
-   aggregate functions automatically in groupby.
+   If an object column includes numerical values such as ``Decimal``
+   objects, it is considered a "nuisance" column. They are automatically
+   excluded from aggregate functions in groupby.
 
-   If you do wish to include decimal or object columns in an aggregation with
+   If you do want to include decimal or object columns in an aggregation with
```

Comment:
I don't see the value of changing wish to want.

Reply:
I'll leave it as it was. 
I was trying to avoid repeating the word too much. IMO when you read the whole document at once it's more engaging if there is variety.